templates/components/agents/typescript/form_filling/attachments/sec_10k_template.csv

```diff
@@ -0,0 +1,17 @@
+﻿Parameter,2023 Apple (AAPL),2023 Tesla (TSLA)
```

Comment:
@marcusschiesser  Due to the different of file structure in Nextjs and Express so i have to move the example CSV file to `attachments` folder. Please let me know if you have better naming or idea.